docs/UserGuide.md

```diff
@@ -2,7 +2,9 @@
 
 ## Introduction
 
-{Give a product intro}
+Financial Planner is a Command Line Interface (CLI) application for managing your finances conveniently. 
```

Comment:
UserGuide looks very good!

Shall we add a table of contents?

Reply:
added!